src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -85,6 +85,8 @@ public BindingWorkflow(IHost host, BindCommandArgs bindingArgs)
             this.nugetBindingOperation = nugetBindingOperation;
         }
 
+        internal /*for testing*/ INuGetBindingOperation NuGetBindingOperation {  get { return this.nugetBindingOperation; } }
```

Comment:
This isn't ideal: it's only there so the unit test can check that the workflow picked the right NuGet behaviour. I might create a factory class later for this and the solution binding operation, but it isn't critical at the moment.